packages/graphql/src/schema/mutation.ts

```
@@ -1703,6 +1705,108 @@ export const Mutation = builder.mutationType({
+      resolveActivityLogEntry: t.withAuth(asUserFullAccess).field({
+        nullable: true,
+        type: ActivityLogEntry,
+        args: {
+          id: t.arg.int({ required: true }),
+        },
+        resolve: async (_, args, ctx) => {
+          return null
+
+          // TODO: implement resolveActivityLogEntry
+          // how to do permissions smartly here? permission on the source object ADMIN or higher?
+          return await SharingService.resolveActivityLogEntry(args, ctx)
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)